src/derived.rs

```diff
@@ -101,6 +112,10 @@ where
         true
     }
 
+    fn memoized_value_eq(_old_value: &Q::Value, _new_value: &Q::Value) -> bool {
+        false
```

Comment:
`VolatileValue`s now always return `false` (and therefore do not need an `Eq` bound), but you mentioned originally:

> maybe we want to have a new policy ("untracked") where we do the equality comparisons

I'd like to have a little more discussion on what you mean by that - perhaps in the [Zulip](https://salsa.zulipchat.com/#narrow/stream/146365-good-first-issue/subject/.2356.20remove.20.60Eq.60.20bounds) topic?

Reply:
Yeah, we can defer that for the purposes of this PR I think anyway.